src/processing/dom/index.js

```diff
@@ -216,6 +216,11 @@ export default class DomProcessor {
         this.adapter.setAttr(el, 'autocomplete', 'off');
     }
 
+    _processContentSecurityPolicy (el) {
+        if (this.adapter.getAttr(el, 'http-equiv').toLowerCase() === 'content-security-policy')
+            this.adapter.removeElement(el);
```

Comment:
> @inikulin right, you remove the array item in the foreach loop

we use `for` loop for selector patterns - https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/processing/dom/index.js#L171
We don't remove the array item in the foreach loop